pkg/controller/argocd/keycloak.go

```diff
@@ -228,18 +242,6 @@ func getKeycloakDeploymentConfigTemplate(cr *argoprojv1a1.ArgoCD) *appsv1.Deploy
 				},
 			},
 			Triggers: appsv1.DeploymentTriggerPolicies{
-				appsv1.DeploymentTriggerPolicy{
```

Comment:
Keycloak deployment config **was** configured with "Image Change" and "Config Change" triggers. 

Deployment Config when configured with both "Image Change" and "Config Change", "Image Config" takes the precedence.

Keycloak/RHSSO image streams are not available OOTB on Disconnected OpenShift clusters because of which the replication controller does not come up. It also does not make much sense to have an "Image Change" trigger in this case as we dont want to create a new Keycloak pod for every Image stream tag update.

https://docs.openshift.com/container-platform/3.5/dev_guide/deployments/basic_deployment_operations.html#triggers

We can also try using a Deployment instead of Deployment Config in **future** but we have to give over some advantages of Deployment config like 
https://developers.redhat.com/blog/2019/09/20/using-red-hat-openshift-image-streams-with-kubernetes-deployments#benefits_of_image_streams